ones_and_zeros.js

```diff
@@ -0,0 +1,6 @@
+// https://www.codewars.com/kata/ones-and-zeros/train/javascript
+
+const binaryArrayToNumber = arr => {
+    const toBinary = arr.toString(2).replace(/[^0-9\.]+/g, "");
+    return parseInt(toBinary, 2);
```

Comment:
Very clever :smile: However, how about actually doing the math and not relying on built-in functions :smile: ? I think that was the purpose of this task - to get you to learn binary